code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm

```diff
@@ -142,6 +142,16 @@
 
 /mob/living/simple_animal/hostile/abnormality/despair_knight/breach_effect(mob/living/carbon/human/user)
 	..()
+	if(!isnull(blessed_human))
+		blessed_human.cut_overlay(mutable_appearance('ModularTegustation/Teguicons/tegu_effects.dmi', "despair", -MUTATIONS_LAYER))
+		UnregisterSignal(blessed_human, COMSIG_LIVING_DEATH)
+		UnregisterSignal(blessed_human, COMSIG_HUMAN_INSANE)
+		blessed_human.physiology.red_mod /= 0.5
+		blessed_human.physiology.white_mod /= 0.5
+		blessed_human.physiology.black_mod /= 0.5
+		blessed_human.physiology.pale_mod /= 2
+		to_chat(blessed_human, "<span class='danger'><i>Feelings of loneliness overtake you as you no longer feel protected.</i></span>")
```

Comment:
We should probably change blessed into a status effect...

Reply:
I'll do that as well.